app/server.js

```diff
@@ -0,0 +1,32 @@
+// native modules
+const fs = require('fs');
+const http = require('http');
+const https = require('https');
+
+// application
+const logger = require('winston');
+const nconf = require('../config');
+const sslcert_key = '../config/sslcert/server.key';
+const sslcert_crt = '../config/sslcert/server.crt';
+
+function createServer(app)
+{
```

Comment:
brackets for functions should start from the same line they are introduced
eg. `function foo() {`